bubble_sort/Python/bubble_sort.py

```diff
@@ -0,0 +1,13 @@
+def bubbleSort(arr):  # Function that performs the bubbleSort
+
+    n = len(arr)  # Finding the length of the array using the len function
+
+    for i in range(n):  # Traverse through all array elements
+        for j in range(0, n-i-1):  # Traverse the array from 0 to n-i-1
+            if arr[j] > arr[j+1]:  # Swap if the element found
+                arr[j], arr[j+1] = arr[j+1], arr[j]  # greater than next element
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpjsnyunsd/bubble_sort/Python/bubble_sort.py
+++ b/tmp/tmpjsnyunsd/bubble_sort/Python/bubble_sort.py
@@ -6,6 +6,8 @@
         for j in range(0, n-i-1):  # Traverse the array from 0 to n-i-1
             if arr[j] > arr[j+1]:  # Swap if the element found
                 arr[j], arr[j+1] = arr[j+1], arr[j]  # greater than next element
+
+
 arr = [12, 96, 24, 48, 36, 60, 84, 72]    # Declaring a sample array
 bubbleSort(arr)  # Calling the bubbleSort function and passing the array
 print("Sorted array is:")  # Printing the array after sorting it
```